src/AutoClosingPairs.re

```diff
@@ -1,72 +1,92 @@
 open EditorCoreTypes;
 
-module AutoClosingPair = {
+type passThroughCharacter = string;
+
+module AutoPair = {
   type t = {
     opening: string,
     closing: string,
   };
-
-  let create = (~closeBefore=[], ~opening, ~closing, ()) => {
-    let hash = Hashtbl.create(16);
-    List.iter(item => Hashtbl.add(hash, item, true), closeBefore);
-
-    {opening, closing};
-  };
 };
```

Comment:
The way you're using this suggests it shouldn't be contained in a separate module. If you just have it at the top-level as `type pair = ...` you could avoid all the `AutoPair.t` annotations.